src/test/resources/mule_sshd_config

```diff
@@ -1,2 +1,2 @@
-KexAlgorithms diffie-hellman-group14-sha1,diffie-hellman-group-exchange-sha512@ssh.com,diffie-hellman-group1-sha1
+KexAlgorithms diffie-hellman-group14-sha1,    diffie-hellman-group-exchange-sha512@ssh.com,diffie-hellman-group1-sha1
```

Comment:
This change is probably an unwanted change.

Reply:
Actually this file is being referenced in both junits and munits written.